### PR TITLE
Desktop portfolio version

### PR DESCRIPTION
**Pull request by MarkoKermi**
Creating Portfolio Desktop version: 

1. Additional correction in CSS based on new experience (reconstruction based on class names)
2. Adding @media queries in CSS.
3. Additional correction in HTML file (adding more understandable class names)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -396,3 +444,157 @@ textarea {
   margin-top: 32px;
   margin-bottom: 8px;
 }
+
+@media screen and (min-width: 738px) {
```
Comment (Yazino12) [inline style.css]: - [ ] Thumps up for adding the media queries for the desktop screens 👍🏽, yet you need to have a larger breakpoint for this milestone as the preferred Desktop breakpoints are in the range of `768px` or wider. Kindly change your breakpoints to `768px`. _please note that it doesn’t have to be exactly_ `768px`, _any value between_ **768** _and_ **1440** _is ok_ 👌🏽.

File: index.html [hunk 2]
```diff
@@ -102,25 +112,26 @@ <h1 class="name">Multi-Post Stories</h1>
...
+        <section class="linkButtons">
+          <ul class="ddm">
+            <li class="ddb"><button class="link">html</button></li>
+            <li class="ddb"><button class="link">css</button></li>
+            <li class="ddb"><button class="link">javaScript</button></li>
+          </ul>
+        </section>
+        <div class="seeMore">
+          <button class="more">See Project</button>
+        </div>
```
Comment (Yazino12) [inline index.html:131]: - [ ] You did well organizing this list 👊🏽, but they have less padding according to the Figma design template as you can see in the snapshots below. Kindly fix this 👇🏽
- [ ] The `See Project` button has less width in all screen sizes, which is causing the text to wrap. Kindly give it the right width so the text won't wrap 👇🏽

### Figma design template
image

### Your design
image

File: index.html [hunk 3]
```diff
@@ -102,25 +112,26 @@ <h1 class="name">Multi-Post Stories</h1>
...
+        </section>
+        <section class="linkButtons">
+          <ul class="ddm">
+            <li class="ddb"><button class="link">html</button></li>
+            <li class="ddb"><button class="link">css</button></li>
+            <li class="ddb"><button class="link">javaScript</button></li>
+          </ul>
+        </section>
+        <div class="seeMore">
+          <button class="more">See Project</button>
```
Comment (Yazino12) [inline index.html:130]: - [ ] To improve your portfolio's design and user navigation I suggest you change the current enabled state of your buttons to match the given design template 🛠

### Figma Design Template
image

### Your Design
image

File: index.html [hunk 4]
```diff
@@ -12,86 +12,96 @@
...
+              <li>About</li>
+              <li>Contact</li>
+            </ul>
       </nav>
     </header>
-    <div class="container">
-      <div class="Personal_info">
+    <section class="background_img">
+      <div class="headline">
         <h2 class="Name_info">I'm Marko Kermichiev Glad to see you!</h2>
```
Comment (Yazino12) [inline index.html]: - [ ] Please make sure that your main header is divided into two lines as you can see in the design below 👇🏽. **_You can use `<br>` line breaker_**

### Figma design
image

### Your design
image

File: index.html [hunk 5]
```diff
@@ -12,86 +12,96 @@
...
         <p class="More_info">
           I'm a software developer! I can help you build a product , feature or
           website Look through some of my work and experience! If you like what
           you see and have a project you need coded, don't hesitate to contact
           me.
+          <p class="connect">LET'S CONNECT</p>
```
Comment (Yazino12) [inline index.html:40]: - [ ] Please make sure that your components align with each other as in the given [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) 📃. This helps the overlook of your portfolio and makes the elements clear and visible 👊🏽.

image

**Review by Yazino12 — CHANGES_REQUESTED** [review]
## Hi @MarkoKermi 🖐🏾,

You have done a great job implementing most of the desktop version requirements 🎯, but there are still some changes that you need to make before moving to the next project 🚀🚀.

### To Highlight:
- Correct use of GitHub flow ✔
- Descriptive pull request title and summary ✔
- Descriptive Readme

# STATUS: REQUIRED CHANGES ♻♻♻
THERE
_Check the comments under the review 👇🏽👇🏽👇🏽._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @MarkoKermi , I'm @MaganaAsiati , your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏

**Pull request merged**